pixels-common/src/main/resources/pixels.properties

```diff
@@ -188,6 +188,11 @@ executor.intra.worker.parallelism=8
 executor.function.service=lambda
 # which method will be used for data exchanging, batch or stream
 executor.exchange.method=batch
+# the first port used by pipeline worker
+executor.worker.exchange.port=50010
+# the ip and port of worker coordinator
+executor.worker.coordinator.ip=10.168.4.4
```

Comment:
We already have `worker.coordinate.server.host` and `worker.coordinate.server.port` in pixels.properties.